resources/js/Shared/Errors.vue

```diff
@@ -8,18 +8,38 @@
 
 <template>
   <div>
-    <div v-if="errors.length > 0" class="border-red ba br3 pa3" :class="classes">
+    <div v-if="dataerror || exception" class="border-red ba br3 pa3" :class="classes">
       <p class="mv0 fw6">{{ $t('app.error_title') }}</p>
-      <p class="mb0">{{ errors[0] }}</p>
-      <template v-if="errors[1]">
-        <p v-for="item in errors[1]" :key="item.index" class="mb0 mt2">{{ item[0] }}</p>
+      <template v-if="dataerror">
+        <p v-if="flatten[0] != 'The given data was invalid.'" class="mb0">
```

Comment:
We need to translate this.

Reply:
I think it comes from Laravel, I don't know if it's translated

Reply:
Confirmed: it's not translated: https://github.com/laravel/framework/blob/8.x/src/Illuminate/Validation/ValidationException.php#L56